### PR TITLE
Compliance: Paper wallet printing

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
[Private key printing](https://trello.com/c/A9y5mQE1/808-private-key-printing)

File: lib/routes.js [hunk 1]
```diff
@@ -155,6 +155,15 @@ function postTx (req, res, next) {
     .catch(next)
 }
 
+function createWallet (req, res, next) {
+  const pi = plugins(req.settings, req.deviceId)
+  pi.newAddress(req.tx)
```
Comment (joshmh) [inline lib/routes.js]: What about the private key?

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/routes.js [hunk 2]
```diff
@@ -367,6 +367,7 @@ app.post('/phone_code', getCustomerWithPhoneCode)
 app.patch('/customer/:id', updateCustomer)
 
 app.post('/tx', postTx)
+app.post('/tx/createWallet', createWallet)
```
Comment (joshmh) [inline lib/routes.js]: Where is the ``createWallet`` function?

File: migrations/1552064796155-tx-paper-wallet.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const db = require('./db')
+
+exports.up = function (next) {
+  var sql = [
+    'ALTER TABLE cash_in_txs ADD COLUMN is_paper_wallet boolean null DEFAULT false',
```
Comment (joshmh) [inline migrations/1552064796155-tx-paper-wallet.js:7]: Is this being used?

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/routes.js (hunk 2 above)
Comment (ghost) [reply lib/routes.js]: That one escaped me. Removed.

File: migrations/1552064796155-tx-paper-wallet.js (hunk 3 above)
Comment (ghost) [reply migrations/1552064796155-tx-paper-wallet.js:7]: Yes, the [migration script](https://github.com/lamassu/lamassu-server/blob/dev/bin/lamassu-migrate) goes through all the scrips in the migration directory.

Comment (joshmh) [reply migrations/1552064796155-tx-paper-wallet.js:7]: I mean is the ``is_paper_wallet`` field ever being set?

Comment (ghost) [reply migrations/1552064796155-tx-paper-wallet.js:7]: Yes, it is. [See here](https://github.com/lamassu/lamassu-machine/blob/449a6cae14d97aa875281cd72372bd040a554699/lib/brain.js#L1775).

**Pull request merged**